Update CONTRIBUTING.md (#250)
openMF/mobile-wallet: .github/CONTRIBUTING.md

When a repository is cloned, it has a default remote called `origin` that points to your fork on GitHub, not the original repository it was forked from. To keep track of the original repository, you should add another remote named `upstream`:

The easiest way is to use the https URL:

`git remote add upstream https://github.com/openMF/mobile-wallet.git`

or if you have ssh set up you can use that URL instead:

`git remote add upstream git@github.com:openMF/mobile-wallet.git`

## Commit Style Guide

>  Mifos Mobile Wallet Android application has set of [Commit Style Guidelines](https://github.com/openMF/mobile-wallet/wiki/Commit-style-guide). we strictly following this guideline to track every change, any bug fixes, any enhancement, and any new feature addition. we are suggesting you, please follow these guidelines to help us managing every commit.

Reviewer: ```suggestion
 Mifos Mobile Wallet Android application have a set of [Commit Style Guidelines](https://github.com/openMF/mobile-wallet/wiki/Commit-style-guide). We strictly follow this guideline to track every change, any bug fix, any enhancement, and any new feature addition. We are suggesting you to please follow these guidelines to help us managing every commit.
```
